Starforge/Editor/ToolManager.cs

```diff
@@ -35,26 +33,51 @@ public static class ToolManager {
         public static int FGTileset;
 
         static ToolManager() {
-            Tools = new Dictionary<ToolType, Tool>()
-            {
-                [ToolType.TileBrush] = new TileBrushTool(),
-                [ToolType.TileRectangle] = new TileRectangleTool(),
-                [ToolType.Entity] = new EntityTool()
-            };
-
-            SelectedTool = Tools[ToolType.TileBrush];
+            Tools = new Dictionary<string, Tool>();
+
             SelectedLayer = ToolLayer.Foreground;
 
             BGTileset = 0;
             FGTileset = 0;
         }
 
         public static void Render() {
+            if (SelectedTool == null)
+                SelectedTool = Tools["TileBrush"];
```

Comment:
Again, please either one line or with brackets
```suggestion
            if (SelectedTool == null) SelectedTool = Tools["TileBrush"];
```